.github/workflows/release.yaml

```diff
@@ -100,6 +102,11 @@ jobs:
           POSTHOG_API_KEY: ${{ secrets.POSTHOG_API_KEY }}
           POSTHOG_ENDPOINT: ${{ secrets.POSTHOG_ENDPOINT }}
 
+      - name: Build SLSA Attestation
+        uses: actions/attest-build-provenance@v2
+        with:
+          subject-path: dist/**/*
```

Comment:
so this is going to attest the binaries? Should we also attest the container images? It seems to be possible with this [attestor](https://github.com/actions/attest-build-provenance?tab=readme-ov-file#container-image)

The container image format can be found in the go-releaser file and are basically these three

![image](https://github.com/user-attachments/assets/6ce7b7ef-af6d-4def-b53b-da4bf317681b)

Reply:
Yes, finally all elements should be included in attestation.